web-react/src/pages/directory/update/CloseChurchMutations.js

```diff
@@ -57,29 +57,27 @@ export const MAKE_COUNCIL_INACTIVE = gql`
 `
 
 export const MAKE_STREAM_INACTIVE = gql`
-  mutation CloseDownStream($id: ID!, $leaderId: ID!) {
-    CloseDownStream(streamId: $id, leaderId: $leaderId) {
```

Comment:
Please help with an explanation of why `leaderId` is being deleted?

Reply:
That was the fix @dabick14  proposed to fix the error when closing down a stream